docs/source/essentials.md

```diff
@@ -6,11 +6,19 @@ description: 'Things you should understand to get the most out of Rover'
 
 ## Graph vs Subgraph
 
-> TODO
+Rover uses the terms `graph` and `subgraph` to differentiate between a monolithic,
+traditional GraphQL endpoint and a federated graph following Apollo's federation
+spec, respectively. Generally, when working with federated graphs, you will be running
+operations based on a single _subgraph_ (and thus using the `subgraph` command), 
+rather than on the whole composed graph.
```

Comment:
```suggestion
Rover uses the terms `graph` to refer to traditional, monolithic graphs, and `subgraph` to refer to each "subgraph" that is a part of a larger [federated graph](link to federation docs). Generally, when working with federated graphs, you will be running
operations based on a single _subgraph_ (and thus using the `subgraph` command), 
rather than on the whole composed graph.
```

maayybbe a bit better though I still dont love it - just trying to work on that first sentence since there's a bunch to think about.

Reply:
Definitely better! I added that link. Feel free to iterate more if you see any better way to word it!